### DISCUSSION TITLE
Hackaton - Busco equipo

### DISCUSSION THREAD

**Discussion by Gabriand** [Launch-X-Latam/MisionFrontEnd#540] [Team Posts]
Hola a todos y una disculpa por las molestias con los mensajes a los que les notifican, acabo de ver la clase del día lunes y ya entiendo el porque de los grupos, hay alguna vacante disponible :(. 
Estaría muy agradecido si me permiten estar en algún grupo :3.

**Comment by edyy-1010** [comment]
me pueden pasar el link de la clase para mirar que toca hacer y si me pueden agregar al un grupo  también

**Comment by Gabriand** [comment]
Hola Eddy, cual link de la clase necesitas para pasarte?

**Comment by edyy-1010** [comment]
la de backend , es que no la tengo y no estuve esa vez tampoco me llego la invitación a la clase

**Comment by Gabriand** [comment]
https://web.microsoftstream.com/channel/128255fd-081d-48b9-8ba9-479008367142

**Comment by edyy-1010** [comment]
gracias  si me puedes agregar al grupo que tengas quedo muy agradecido

**Comment by Gustavo1770** [comment]
Holaaa! Me pueden agregar a su equipo? Este es mi usuario lx03022
